Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -62,14 +62,50 @@ WarpX::ComputeSpaceChargeField (bool const reset_fields)
 
     if (do_electrostatic == ElectrostaticSolverAlgo::LabFrame) {
         AddSpaceChargeFieldLabFrame();
-    } else {
+    }
+    else {
+        bool calculate_background_field = false;
         // Loop over the species and add their space-charge contribution to E and B
```

Comment:
Could you add a comment saying that this space-charge solve does not include the static electric field from `phi` at the boundaries? (and that that part of the electric field is added further below, by a separate function call)